### PR TITLE
[3JH] Live documentation for saas loader page #332

### PR DESCRIPTION
**Pull request by Loris-Reggiani**
# Description

This is a static page that explains how to use the saas page, with screenshots and examples, so that user easily understand what they should do.

# Changes include

- [ ] Accueil.tsx
- [ ] Generated payload.gif

### CODE REVIEW

**Comment by djnnvx** [comment]
@Loris-Reggiani there is a conflict in Acceuil.tsx. Can you please fix ? Thank you

**Comment by djnnvx** [comment]
image

there are still conflicts

File: web/src/assets/BasicShellcode.gif
Comment (claraxuxu) [inline web/src/assets/BasicShellcode.gif:1]: Can you zoom the website in a normal size to better show the steps please? thx

File: web/src/assets/Generate payload.gif
Comment (claraxuxu) [inline web/src/assets/Generate payload.gif:1]: same here

File: yarn.lock
Comment (claraxuxu) [inline yarn.lock:1]: this should not be pushed

**Review by claraxuxu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**